pyproject.toml

```diff
@@ -9,7 +9,8 @@ readme = "README.md"
 authors = [ "Features & Labels <hello@fal.ai>" ]
 packages = [
         { include = "fal", from = "src" },
-        { include = "faldbt", from = "src" }
+        { include = "faldbt", from = "src" },
+        { include = "_fal_testing", from = "src" }
```

Comment:
ATL: Why are we including this in src?

Reply:
I actually don't have a preference, didn't think about it much but now considering putting this into `tests/` makes sense.

Reply:
Yes. Otherwise, we can't use it in our scripts, and generally, it is not an uncommon practice to package these as well.